static/js/cf-frontend.js

```diff
@@ -116,7 +109,17 @@ function initAceEditor() {
   // don't do real-time syntax checks:
   // https://github.com/ajaxorg/ace/wiki/Syntax-validation
   s.setOption("useWorker", false);
-  //initial blank input field with template
+
+
+  var TEMPLATE_CHECKER = 'nullness';
+  var JAVA_BLANK_TEMPLATE = 'import org.checkerframework.checker.nullness.qual.Nullable;\n\
+    class YourClassNameHere {\n\
+      void foo(Object nn, @Nullable Object nbl) {\n\
+	nn.toString(); // OK\n\
+	nbl.toString(); // Error\n\
+     }\n\
+  }';
+
   if ($.trim(pyInputGetValue()) === '') {
```

Comment:
What if a user encoding an url with empty content but just indicate a specific type system? In this case this code will override the specific type system to the default `nullness`.

Maybe it would be better to check the editor content and type system separately?